### PR TITLE
Support Symfony 7.0

### PR DESCRIPTION
**Pull request by jerowork**
# Description
Since Symfony 7.0 is released, it's time to support here as well.

### CODE REVIEW

**Review by ruudk — APPROVED** [review]

**Comment by jerowork** [comment]
@mcg-web could you trigger the GH action for this to see if all passes 🙇‍♂️

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,7 +26,9 @@ jobs:
                     - 'lowest'
                     - 'highest'
                 remove-dependencies: [ '' ]
-
+                include:
+                  - php-version: '8.2'
+                    symfony-version: '7.0.*'
```
Comment (jerowork) [inline .github/workflows/ci.yml:31]: As SF7.0 is not applicable for PHP 8.1, added a configuration on top of the matrix, to only run SF7.0 for PHP8.2
@see https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations

Comment (jerowork) [reply .github/workflows/ci.yml:31]: @mcg-web Could you re-trigger the GH action one more time? :)

**Comment by jerowork** [comment]
@mcg-web Thank you for triggering the previous GH actions. Ready for review 🎉

**Comment by ruudk** [comment]
@mcg-web could you fix the "workflow awaiting approval" rule? So that it just works all the time. This is super annoying.

**Review by mcg-web — APPROVED** [review]
Thank you @jerowork !

**Comment by jerowork** [comment]
@mcg-web do you already have ideas when a new release is planned, including this change? Thanks!

**Pull request merged**